services/QuillLMS/app/assets/stylesheets/pages/assign_a_new_activity.scss

```diff
@@ -145,3 +164,69 @@
     margin-right: 16px;
   }
 }
+
+.evidence-header {
+  margin-top: 40px;
+  margin-bottom: 32px;
+}
+
+.suggested-activities-table {
+  border: 1px solid #E0E0E0;
+  border-radius: 6px;
+  tr {
+    background-color: white;
+    padding-top: 12px;
+    padding-bottom: 2px;
+    width: $sitewidepagewidth;
```

Comment:
Does this just become scrollable when the page is narrower? We might want some kind of media query here to stop showing it entirely if it becomes nonfunctional at a certain screenwidth.